README.md

```diff
@@ -37,7 +38,13 @@ You may consider to visit [Cygnus Quick Start Guide](doc/quick_start_guide.md) b
 [Top](#top)
 
 ##<a name="section2"></a>Basic operation
-###<a name="section2.1"></a>Installation (CentOS/RedHat)
+###<a name="section2.1"></a>Hardware requirements 
+*  RAM: 1 GB, specially if abusing of the batching mechanism.
+*  HDD: A few GB may be enough unless the channel types are configured as FileChannel type.
```

Comment:
Fixed in <https://github.com/telefonicaid/fiware-cygnus/commit/69bad2f614891f36366db42e5b3636c2d0b591d0>